forte2/orbitals/aset.py

```diff
@@ -209,8 +213,8 @@ def _make_fragment_projector(self):
         X_mm : ndarray
             The metric-orthogonalizer (S_mm^1/2).
         """
-        # 1. Compute minAO overlap S_mm
-        S_mm = ints.overlap(self.system.minao_basis)
+        # 1. Compute AO overlap S_mm
```

Comment:
I suppose we use S_mm because before it indicated the overlap for the minao-minao block. Can you please change the notation to S_AA or S_ff for the fragment projection?